src/rust/engine/remote_provider/remote_provider_traits/src/lib.rs

```diff
@@ -12,9 +12,20 @@ use remexec::ActionResult;
 use tokio::fs::File;
 use tokio::io::{AsyncSeekExt, AsyncWrite};
 
+// TODO: this is duplicated with global_options.py, it'd be good to have this be the single source
+// of truth.
+#[derive(Clone, Copy, Debug, strum_macros::EnumString)]
+#[strum(serialize_all = "kebab-case")]
+pub enum RemoteProvider {
```

Comment:
This enum is duplicated in both Rust and Python, but I think that happens elsewhere too (e.g. `CacheContentBehavior`), so it seems tolerable.